modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -23,45 +23,42 @@ $ oc adm node-logs --role=master --path=openshift-apiserver/
 .Example output
 [source,terminal]
 ----
-ip-10-0-140-97.ec2.internal audit-2019-04-09T00-12-19.834.log
-ip-10-0-140-97.ec2.internal audit-2019-04-09T11-13-00.469.log
-ip-10-0-140-97.ec2.internal audit.log
-ip-10-0-153-35.ec2.internal audit-2019-04-09T00-11-49.835.log
-ip-10-0-153-35.ec2.internal audit-2019-04-09T11-08-30.469.log
-ip-10-0-153-35.ec2.internal audit.log
-ip-10-0-170-165.ec2.internal audit-2019-04-09T00-13-00.128.log
-ip-10-0-170-165.ec2.internal audit-2019-04-09T11-10-04.082.log
-ip-10-0-170-165.ec2.internal audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit-2021-03-09T00-12-19.834.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit-2021-03-09T00-11-49.835.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit-2021-03-09T00-13-00.128.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit.log
 ----
 
-.. View the {product-title} API server log for a specific master node and timestamp or view all the logs for that master:
+.. View the OpenShift API server log for a specific master node and timestamp or view all the logs for that master:
 +
 [source,terminal]
 ----
-$ oc adm node-logs <node-name> --path=openshift-apiserver/<log-name>
+$ oc adm node-logs <node_name> --path=openshift-apiserver/<log_name>
 ----
 +
 For example:
 +
 [source,terminal]
 ----
-$ oc adm node-logs ip-10-0-140-97.ec2.internal --path=openshift-apiserver/audit-2019-04-08T13-09-01.227.log
+$ oc adm node-logs ci-ln-m0wpfjb-f76d1-vnb5x-master-0 --path=openshift-apiserver/audit-2021-03-09T00-12-19.834.log
```

Comment:
You could include some text before both of the example commands or split the whole thing into two bullet points. This way you can describe what each example is doing in turn.

The same applies for the other steps that include multiple example commands adjacently.